corehq/apps/reports/templates/reports/partials/save_reports_modal.html

```diff
@@ -26,7 +26,16 @@ <h3 data-bind="text: modalTitle"></h3>
                     </div>
                 </div>
 
-                <div data-bind="if: date_range()">
+                <div data-bind="visible: has_ucr_datespan">
+                    <div class="control-group">
+                        <label class="control-label">{% trans "Date Filter" %}</label>
+                        <div class="controls">
+                            <select data-bind="options: datespan_filters, value: datespan_slug"></select>
```

Comment:
would be nice to display filter names (instead of slugs) to the user, but I couldn't figure out a way to specify option-value pairs with KO.  Easy to do with crispy forms, but willing to punt on this for now since moving this modal to crispy would take a bit of time.